kolibri/utils/file_transfer.py

```diff
@@ -122,14 +122,16 @@ def _get_chunked_file_dirs(self):
             for dir in dirs:
                 if dir.endswith(CHUNK_SUFFIX):
                     yield os.path.join(root, dir)
+                    # Don't continue to walk down the directory tree
+                    dirs.remove(dir)
 
     def _get_chunked_file_stats(self):
         stats = {}
         for chunked_file_dir in self._get_chunked_file_dirs():
             file_stats = {"last_access_time": 0, "size": 0}
-            for file in os.listdir(chunked_file_dir):
-                file_path = os.path.join(chunked_file_dir, file)
-                if os.path.isfile(file_path):
+            for dirpath, _, filenames in os.walk(chunked_file_dir):
+                for file in filenames:
+                    file_path = os.path.join(dirpath, file)
```

Comment:
Curious how this is accounting for the diskcache differently?

Reply:
It's now doing an os.walk of the entire ChunkedFile dir (including the directory that diskcache is using) so it's enumerating all the files.

Previously, it was only listing the files in the top directory, which was all the chunk files, plus the directory used for diskcache (which it was ignoring because it was a directory).